src/util/get-update-command.ts

```diff
@@ -24,10 +24,76 @@ async function isYarn(): Promise<boolean> {
   return !('_id' in pkg);
 }
 
+async function getConfigPrefix() {
+  const paths = [
+    process.env.npm_config_userconfig || process.env.NPM_CONFIG_USERCONFIG,
+    join(process.env.HOME || '/', '.npmrc'),
+    process.env.npm_config_globalconfig || process.env.NPM_CONFIG_GLOBALCONFIG
+  ].filter(Boolean);
+
+  for (const configPath of paths) {
+    if (!configPath) {
+      continue;
+    }
+
+    const content = await readFile(configPath)
```

Comment:
Don't we want to try/catch this so that if one of the paths fails then the for loop continues processing?

Reply:
@TooTallNate it does, check the lines below, there is a `.catch`